addons/survey/views/survey_survey_views.xml

```diff
@@ -281,6 +370,10 @@
                     domain="[('my_activity_date_deadline', '=', context_today().strftime('%Y-%m-%d'))]"/>
                 <filter invisible="1" string="Upcoming Activities" name="activities_upcoming_all"
                     domain="[('my_activity_date_deadline', '&gt;', context_today().strftime('%Y-%m-%d'))]"/>
+                <group expand="0" string="Group By">
+                    <filter string="Responsible" name="group_by_responsible"
+                        domain="[]" context="{'group_by': 'user_id'}"/>
```

Comment:
I keep seeing that `domain="[]"` thing in group by filters
Is that actually necessary? I think we tested with Stéphane and it's not.
I find it a bit confusing to specify it if not necessary.